src/frontends/tensorflow_lite/src/op_table.cpp

```diff
@@ -61,7 +61,7 @@ std::map<std::string, CreatorFunction> get_supported_ops() {
         // CONV_3D
         // CONV_3D_TRANSPOSE
         {"COS", translate_unary<opset10::Cos>},
-        // CUMSUM
+        {"CUMSUM", translate_cumsum_op},
         // CUSTOM
         // DELEGATE
```

Comment:
could you please add layer test for it? https://github.com/openvinotoolkit/openvino/tree/master/tests/layer_tests/tensorflow_lite_tests